File: CRUD/PHP/Ajax V3/templates/cuenta/cambiar_usuario.php
<?php  

// Written by Ismael Heredia in the year 2017

include_once("../../includes/Funciones.php");

if(!verificarCookie()) {
  exit;
}

$split = base64_decode($_SESSION['uid']);

$result      = explode("@", $split);
$user_login = $result[0];

?>
<div class="panel contenedor panel-default">
<div class="panel-heading">Cambiar Usuario</div>
<div class="panel-body">
  <form action="?cambiar_usuario" method="POST" name="form_cambiar_usuario" id="cambiarUsuarioForm" class="form-horizontal">
	<fieldset>
	  <legend>Datos</legend>
	  <div class="form-group" name="form-group-username">
		<label for="inputNombre" class="col-lg-2 control-label">Usuario</label>
		<div class="col-lg-10">
		  <input class="form-control" id="inputUsuario" placeholder="Ingrese nombre usuario" type="text" name="usuario"  value="<?php echo htmlentities($user_login) ?>" readonly="readonly">
		</div>
	  </div>
	  <div class="form-group" name="form-group-new-username">
		<label for="inputNuevo" class="col-lg-2 control-label">Nuevo usuario</label>
		<div class="col-lg-10">
		  <input class="form-control" id="inputNuevoUsuario" placeholder="Ingrese nuevo usuario" type="text" name="nuevo_usuario">
		</div>
	  </div>
	  <div class="form-group" name="form-group-password">
		<label for="inputActual" class="col-lg-2 control-label">Actual clave</label>
		<div class="col-lg-10">
		  <input class="form-control" id="inputPassword" placeholder="Ingrese clave" type="password" name="clave">
		</div>
	  </div>
	  <div class="form-group">
		  <button type="submit" class="btn btn-primary center-block" name="cambiar_usuario" id="cambiar_usuario"><span class="glyphicon glyphicon-pencil right_space"></span>Cambiar</button>
	  </div>
	</fieldset>
  </form>        
</div>
</div>

File: CRUD/Java usando NetBeans/Aplicacion Web usando MVC Spring con Hibernate/SistemaWebSpringHibernate/SistemaWebSpringHibernate/src/main/java/sistema/models/CambiarUsuario.java
// Written by Ismael Heredia in the year 2016

package sistema.models;

public class CambiarUsuario {
    
    String nombre_usuario;
    String nuevo_usuario;
    String password;

    public String getNombre_usuario() {
        return nombre_usuario;
    }

    public void setNombre_usuario(String nombre_usuario) {
        this.nombre_usuario = nombre_usuario;
    }

    public String getNuevo_usuario() {
        return nuevo_usuario;
    }

    public void setNuevo_usuario(String nuevo_usuario) {
        this.nuevo_usuario = nuevo_usuario;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public CambiarUsuario() {
    }
    
    public CambiarUsuario(String nombre_usuario, String nuevo_usuario, String password) {
        this.nombre_usuario = nombre_usuario;
        this.nuevo_usuario = nuevo_usuario;
        this.password = password;
    }

    @Override
    public String toString() {
        return "CambiarUsuario{" + "nombre_usuario=" + nombre_usuario + ", nuevo_usuario=" + nuevo_usuario + ", password=" + password + '}';
    }
       
}
